src/onegov/org/forms/directory.py

```diff
@@ -757,3 +764,186 @@ def validate_name(self, field: StringField) -> None:
             raise ValidationError(
                 _("An entry with the same name exists")
             )
+
+
+class DirectoryFAQForm(DirectoryBaseForm):
```

Comment:
I think it would be better to add an `is_faq` boolean field at the top and add field dependencies to hide the fields that don't matter for FAQs. We can still have two dedicated views by pre-setting and hiding the field if we wish, although I'm not sure even that is worth it, considering the separate views are just all copies of one another.

Reply:
Yes, let me do so